editor/project_manager/project_dialog.h

```diff
@@ -135,6 +153,32 @@ class ProjectDialog : public ConfirmationDialog {
 	void _reset_name();
 	void _renderer_selected();
 	void _nonempty_confirmation_ok_pressed();
+	void _post_process_project(Mode p_mode, const String &p_path);
+
+	struct ProgressData {
+		Mode mode;
+		String source_path;
+		String target_path;
+		bool create_dir;
+
+		Error error;
```

Comment:
```suggestion
		Error error = OK;
```
Or some appropriate default